OracleTuxedo/salt/samples/bankapp/README.md

```diff
@@ -0,0 +1,54 @@
+
+# Oracle Tuxedo `bankapp` Sample Container Image
```

Comment:
```suggestion
# Oracle SALT bankapp sample application container image
```

Reply:
This should also be on line 1, i.e. remove the empty first line.

Reply:
Removed the first line and modified second line to:
# Oracle Tuxedo SALT bankapp sample application container image